Tests/Rewriters/Strategies/ArgumentStrategies/ArgListIsInArgumentRewriteStrategyTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿//  Copyright (c) rubicon IT GmbH
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"),
+// to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+// FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+// WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+using System;
+using NUnit.Framework;
+using RhinoMocksToMoqRewriter.Core.Rewriters.Strategies.ArgumentStrategies;
+
+namespace RhinoMocksToMoqRewriter.Tests.Rewriters.Strategies.ArgumentStrategies
+{
+  [TestFixture]
+  public class ArgListIsInArgumentRewriteStrategyTests
+  {
+    private readonly IArgumentRewriteStrategy _strategy = new ArgListIsInArgumentRewriteStrategy();
+
+    private readonly Context _context =
+        new Context
+        {
+            //language=C#
+            InterfaceContext = @"void DoSomething (IEnumerable<int> b);",
+            //language=C#
+            MethodContext = @"var mock = MockRepository.GenerateMock<ITestInterface>();"
+        };
+
+    [Test]
+    [TestCase (
+        //language=C#
+        @"mock.DoSomething (Arg<int[]>.List.IsIn (2));",
+        //language=C#
+        @"mock.DoSomething (It.Is<int[]> (param => param.Contains (2)));")]
```

Comment:
I think the original spec may have missed that we have an `It.IsIn(new[]{2})` for single and multi-value list matches. We should check this out before we re-invent the wheel.

Reply:
> I think the original spec may have missed that we have an `It.IsIn(new[]{2})` for single and multi-value list matches. We should check this out before we re-invent the wheel.

In my understanding Arg.List.IsIn and It.IsIn are not the same, Arg.List.IsIn(a, b) checks if a and b are in the list c and It.IsIn(a, b) checks if c is in (a, b)

Reply:
Doh! Thanks.